main.cpp

```diff
@@ -54,6 +59,7 @@ struct CLICallbacks
 		callbacks[cli] = func;
 	}
 	unordered_map<string, function<void(CLIParser &)>> callbacks;
+	uint64_t pad0;
```

Comment:
Yes, I see the slide deck you mention put no-padded in there, but that is not a relevant warning for us.